[2.7.2] Update PSS docs (#479)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md

title: Pod Security Policies
---

:::caution

> Pod Security Policy is only available in Kubernetes until v1.24. Kubernetes v1.25 and above use [Pod Security Standards](pod-security-standards.md).
> :::

Reviewer: We may want to highlight that both co-existed in 1.24 (by default), even though it was [Beta](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) back then.

```suggestion
Pod Security Policy is only available in Kubernetes until v1.24. [Pod Security Standards](pod-security-standards.md) is the built-in alternative, available from Kubernetes v1.24.
:::
```

> [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/) (or PSPs) are objects that control security-sensitive aspects of pod specification (like root privileges).

Reviewer 2: ```suggestion
[Pod Security Policies (PSPs)](https://kubernetes.io/docs/concepts/security/pod-security-policy/) are objects that control security-sensitive aspects of pod specification (like root privileges).
```

> title: Pod Security Policies

Reviewer 3: ```suggestion
title: Pod Security Standards (PSS) & Pod Security Admissions (PSA)
```

See my comment further down about the Rancher GUI.

Also:  It's just hard to talk about either resource without mentioning the other.

It's tempting to pull PSACT content into its own page, since how-to create and edit PSACTs feels like its own topic. That leaves the PSS page with the intro, a section on migration, and a section on Rancher namespace exemptions. Thoughts?

---

:::caution

> Pod Security Policy is only available in Kubernetes until v1.24. [Pod Security Standards](pod-security-standards.md) is the built-in alternative.

Reviewer 3: ```suggestion
Pod Security Policy is only available in Kubernetes until v1.24. [Pod Security Standards](pod-security-standards.md) are the built-in alternative.
```
